components/Form/Cards.js

```diff
@@ -1,17 +1,35 @@
 import styled from "styled-components";
+import Image from "next/image";
+import deleteButton from "/Users/sukhpreet/Desktop/Project/capstone-project/public/images/deleteButton.svg";
 
-export default function Card({ name }) {
+export default function Card({ name, onDelete }) {
   return (
-    <Article>
-      <p>{name}</p>
-    </Article>
+    <>
+      <CardContainer>
+        <HabitContainer>{name} </HabitContainer>
+        <ImageContainer onClick={onDelete}>
+          <Image src={deleteButton} alt="delete your habit" />
+        </ImageContainer>
+      </CardContainer>
+    </>
```

Comment:
You don't need the fragment element (<>) if it only contains one element. You only need it when you want to return 2 or more elements on the same level, since react only allows one element as a return value.